pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/CancerTypeaheadComparator.java

```diff
@@ -14,6 +13,7 @@
 public class CancerTypeaheadComparator implements Comparator<Cancer> {
```

Comment:
Refactored to use StringTypeaheadComparator and avoid code duplication.
Cancer, Drug uses exact same logic to compare & sort the suggestions